src/Symfony/Component/HttpKernel/DataCollector/DataCollectorInterface.php

```diff
@@ -33,13 +36,4 @@
      * @api
      */
     public function collect(Request $request, Response $response, \Exception $exception = null);
```

Comment:
There are 2 reason for this, for one the new DataCollectorInterface doesn't have a collect function any more.
RuntimeDataCollectorInterface replace the old DataCollectorInterface, as I noticed some if not most use the LateDataCollectorInterface and have no use for a collect method.
For that reason I have removed the collect method from DataCollectorInterface.

The other reason is that the new profiler does not require a Request and Response, but to maintain BC I had to leave this in.